doc/source/tutorial/tutorial_IO.ipyml

```diff
@@ -1,18 +1,605 @@
+    
+    ``open_excel`` must be used with the Python keyword ``with``. If the Excel file doesn't exist, the argument ``overwrite_file`` must set to True. The method ``save`` must be called to actually write data in the Excel file. 
+    For example: 
+
+
+- code: |
+    arr = ndtest((2, 2))
+    arr2 = zeros((2, 2))
+    arr3 = ones((2, 2))
+    
+    # to create a new Excel file, argument overwrite_file must be set to True
+    with open_excel('new_excel_file.xlsx', overwrite_file=True) as wb:
+        # add a new sheet 'arr' and dump the array 'arr' in it 
+        wb['arr'] = arr.dump()
+        # add a new sheet 'arr2' and dump the array 'arr2' in it 
+        wb['arr2'] = arr2.dump()
+        # add a new sheet 'arr3' and dump the array 'arr2' in it 
+        wb['arr3'] = arr3.dump()
+        # actually write data in the Workbook
+        wb.save()
+        
+    # the Workbook is automatically closed when getting out the block 
+    # defined by the with statement
+    print('dumping arrays arr, arr2 and arr3 is done')
+
+  id: 6
+
+- markdown: |
+    ##### Specifying Position
+    
+    It is possible to choose the position of the first cell containing the data of the dumped array. This allows to export several arrays in the same sheet: 
+
+
+- code: |
+    arr = ndtest((2, 2))
+    arr2 = zeros((2, 2))
+    arr3 = ones((2, 2))
+    
+    with open_excel('new_excel_file.xlsx') as wb:
+        # add a new sheet 'arrays' and dump the array 'arr' starting at cell 'A1' 
+        wb['arrays'] = arr.dump()
+        # dump the array 'arr2' in sheet 'arrays' starting at cell 'A5' 
+        wb['arrays']['A5'] = arr2.dump()
+        # dump the array 'arr3' in sheet 'arrays' starting at cell 'A9' 
+        wb['arrays']['A9'] = arr3.dump()
+        wb.save()
+
+  id: 7
+
+- markdown: |
+    ##### Specifying Wide VS Narrow format
+    
+    Like with the ``to_excel`` method, it possible to export the data in a ``narrow`` format by setting the argument ``wide`` to False:
+
+
+- code: |
+    arr = ndtest((2, 2))
+    
+    with open_excel('new_excel_file.xlsx') as wb:
+        # dump the array 'arr' in a narrow format: 
+        # one column per axis plus one value column.
+        # Argument value_name can be used to change the name of the 
+        # column containing the data (default name is 'value')
+        wb['narrow'] = arr.dump(wide=False, value_name='data')
+        wb.save()
+        
+    # in the sheet 'narrow', data are written as:
+    # a  | b  | data  |
+    # a0 | b0 | 0     |
+    # a0 | b1 | 1     |
+    # a1 | b0 | 2     |
+    # a1 | b1 | 3     |
+
+  id: 8
+
+- markdown: |
+    ##### Exporting only data
+    
+    To export only data in Excel sheets, you can set argument ``header`` of the method ``dump`` to false or not calling ``dump``:
+
+
+- code: |
+    arr = ndtest((2, 2))
+    
+    with open_excel('new_excel_file.xlsx') as wb:
+        # export only data
+        wb['data_only'] = arr.dump(header=False)
```

Comment:
When users export only data, they are usually in one of two situations:
1. they want to fill data in a "predefined/template file".
2. they want to export data and axes in a specific way. For example, two 2d tables on top of each other with a single "time" axis in columns but potentially blank lines between the two tables.

In both cases, it uses 'precise' position.
1. is probably hard to demonstrate but is **very** frequent so should at least be mentioned. If you can come up with a simple example to demonstrate this, it would be nice. Honestly I don't see how we could make this simple enough and still interesting but I think our users would appreciate if you managed that. Maybe first writing the array with headers, then saying this is the template to fill and updating only the values would be enough, or maybe using the specific formatted sheet done for 2. and updating the values would be more telling.
2. could be demonstrated, and if so, needs to include an example of how to output an axis labels vertically instead of horizontally (sheet[idx].options(transpose=True).value = axis.labels)

Reply:
Concerning the second situation, I'm thinking of an additional column 'description' next to a vertical axis column. What do you think?